spec/unit/config_spec.rb

```diff
@@ -45,7 +45,7 @@
       ChefSpec::SoloRunner.new do |node|
         node.set['ohai']['disabled_plugins'] = [:passwd, "dmi"]
         node.set['chef_client']['config']['log_level'] = ":debug"
-        node.set['chef_client']['config']['ssl_verify_mode'] = ":verify_peer"
+        node.set['chef_client']['config']['ssl_verify_mode'] = ":verify_none"
```

Comment:
I think we should keep the default to verify peer. The rationale is that is what chef client does. Or we can remove this default entirely.